include/xtensor/xaccumulator.hpp

```diff
@@ -68,6 +68,24 @@ namespace xt
         return accumulator_type(std::forward<RF>(accumulate_func), std::forward<IF>(init_func));
     }
 
+    template <std::size_t I, typename ...T>
+    decltype(auto) get(xaccumulator_functor<T...>&& v)
```

Comment:
I think these functions could be more generic so we don't have to define them for every class inheriting from `std::tuple`:

```cpp
template <std::size_t I, tempalte<class...> C, class... T>
decltype(auto) get(C<T...>&& v)
{
    return std::get<I>(static_cast<std::tuple<T...>&&>(v));
}
// etc...
```

These generic versions can be put in `xutils.hpp`.
Also it could be nice to explain why we needed to do that instead of directly use `std::get` (the link to the SO issue might be enough).

Reply:
Done.